azure/resource_groups/app/template.json

```diff
@@ -32,9 +32,33 @@
       "type": "bool",
       "defaultValue": true
     },
+    "appServicePlanInstances": {
+      "type": "int",
+      "defaultValue": 2
+    },
+    "appServicePlanTier": {
+      "type": "string",
+      "defaultValue": "Standard"
+    },
+    "appServicePlanSize": {
+      "type": "string",
+      "defaultValue": "1"
+    },
+    "vspAppServicePlanInstances": {
+      "type": "int",
+      "defaultValue": 1
```

Comment:
Is there a reason for this to have a different default from the web app service plan?